lib/cmds/space_cmds/environment_cmds/create.js

```diff
@@ -0,0 +1,59 @@
+import { getContext } from '../../../context'
+import * as logging from '../../../utils/log'
+import { handleAsyncError as handle } from '../../../utils/async'
+import { createManagementClient } from '../../../utils/contentful-clients'
+import { assertLoggedIn, assertSpaceIdProvided } from '../../../utils/assertions'
+
+export const command = 'create'
+
+export const desc = 'Create an environment'
+
+export const builder = (yargs) => {
+  return yargs
+    .usage('Usage: contentful environment create --name \'Your Environment Name\'')
+    .option('environment-id', {
+      alias: 'e',
+      describe: 'Id of the environment to create',
+      demandOption: true
+    })
+    .option('name', {
+      alias: 'n',
+      describe: 'Name of the environment to create'
+    })
+    .option('space-id', {
+      describe: 'ID of the space the environment will belong to',
```

Comment:
the sentence is correct but `ID of the space **that** the environment will belong to` is better